index.js

```diff
@@ -1,18 +1,68 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = 'orange'
 
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
-
+const image = document.querySelector('#profile img')
+const h2 = document.querySelector('#profile h2')
+const em = document.querySelector('#profile em')
+// console.log(em)
+image.src = traveler.photo 
+image.alt = traveler.name
+h2.textContent = traveler.name
+em.textContent = traveler.nickname
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function(animalSighting) {
+const li = document.createElement('li')
+const p = document.createElement('p')
+const img = document.createElement('img')
+const a = document.createElement('a')
+
+li.dataset.id = animalSighting.id
+
+p.textContent = animalSighting.description
+
+// img.source = animalSighting.photo
+//not showing image - fixed
+img.src = animalSighting.photo
+img.alt = animalSighting.species
+
+a.href = animalSighting.link
+a.target = "_blank"
+a.textContent = `Here's a video about the ${animalSighting.species} species!`
 
+li.append(p, img, a)
+
+const ul = document.querySelector('#animals')
+ul.append(li)
+})
```

Comment:
nice! great work! This approach is actually more efficient than using innerHTML, in terms of time and space complexity. So good choice!